docs/docs/configuration/connecting-to-data-stores/opensearch.md

```diff
@@ -50,40 +50,35 @@ service:
 
 ## Configure Tracetest to Use OpenSearch as a Trace Data Store
 
-You also have to configure your Tracetest instance to make it aware that it has to fetch trace data from Jaeger. 
+You also have to configure your Tracetest instance to make it aware that it has to fetch trace data from OpenSearch. 
 
-Edit your configuration file to include this configuration:
+Make sure you know which Index name and Address you are using. In the screenshot below, the Index name is `traces`, the Address is `http://opensearch:9200`.
 
-```yaml
-# tracetest.config.yaml
+### Web UI
 
-postgresConnString: "host=postgres user=postgres password=postgres port=5432 sslmode=disable"
+In the Web UI, open settings, and select OpenSearch.
 
-poolingConfig:
-  maxWaitTimeForTrace: 10m
-  retryDelay: 5s
+![](https://res.cloudinary.com/djwdcmwdz/image/upload/v1674644099/Blogposts/Docs/screely-1674644094600_svcwp6.png)
```

Comment:
Just out of curiosity: Since we are focusing on `docs/` folder being a central source of documentation, does it make sense to move these images to this structure as well? Or there is a reason to host these images on cloudinary?

Reply:
Might be a question for @adnanrahic